Dockerfile

```diff
@@ -224,7 +238,7 @@ RUN useradd --comment 'Islandora User' --no-create-home -d /var/www/html --syste
     ## BUILD TOOLS
     mkdir /utility-scripts && \
     cd /utility-scripts && \
-    git clone $ISLE_BUILD_TOOLS_REPO -b $ISLE_BUILD_TOOLS_BRANCH && \
+    git clone $ISLE_BUILD_TOOLS_REPO -b $ISLE_BUILD_TOOLS_BRANCH isle_drupal_build_tools && \
```

Comment:
FYI that this change just guarantees that the build tools go into a consistently named repo. There was something different about how macs/linux computers handled capitalization and underscores/dashes.